src/views/Player.vue

```diff
@@ -120,9 +120,11 @@ export default defineComponent({
       () => state.currentQuestionIndex,
       (newValue) => {
         if (newValue == state.questions.length) {
-          state.isScorecardShown = true;
-          calculateScorecardMetrics();
           if (areAllQuestionsSurvey.value) state.isScorecardShown = false;
+          else {
```

Comment:
you can just return out from the function in the `if` condition above and then, you won't need `else` right?